python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py

```diff
@@ -156,6 +163,14 @@ async def configure(self, config):
         self.actuators_to_test = (
             self.m1m3_actuator_ids if config.actuators == "all" else config.actuators
         )
+        if config.ignore_actuators:
+            self.actuators_to_test = list(
+                filter(
+                    lambda actuator: actuator not in config.ignore_actuators,
```

Comment:
lambda functions are not allowed per our development guidelines: https://developer.lsst.io/python/style.html#lambda-should-not-be-used

I would suggest doing something like:

```
self.actuators_to_test = [actuator_id for actuator_id in self.actuators_to_test if actuator_id not in config.ignore_actuators]
```

It is more clear and expressive than the implementation here.